### ISSUE TITLE
Shadow/Shadowmaster Balance adjustent request

### ISSUE THREAD

**Issue by Voltem0** [gottsch/gottsch-minecraft-Dungeon-Denizens#4]
Hello

I tested out this mod and i am generally very happy with the D&D/fantasy aestetics of it.
However when actually running it there were some distinct problems with the shadow monsters that stood out to me and i wanted to bring up for your consideration

First off, the shadowmasters are too common. Its a (mini)boss style encounter that if you are unprepared for it will really mess you up. Its very powerful and has a great amount of HP.
However its spawn rates seem to be far too high. I could fly through a cave system in creative mode and count groups of 5-10 shadowmasters in a single cave. For its sheer strength the creature is too common and the spawnrates should be adjusted (lowered).
Perhaps to something like the gazer's, because i was unable to find that creature spawning naturally.
I would also recommend you give the shadowmancer a distinct sound so that players are aware its around and be not (as) surprised by it when exploring caves. Perhaps a remix of the blaze sound if you dont want to make your own.

Secondly i take issue with the shadowmancer only really being defeatable by gold tools and armor due to its massive damage resistance and crippling darkness effect.
That is just not a very fun way of designing creatures, especially since the player has no way of finding out its weakness without checking this mod's page.
Its fine if the shadows have a weakness to gold, but it shouldnt be that that is the only way to reasonably beat them. Ideally it would be that you can defeat a shadow(master) with an iron sword, you can just do so much more easily by using gold equipment.
Also i wanted to ask, are those monsters considered 'undead'? Do they interact with the 'smite' enchantment?

TL;DR: i request the spawn rates for the shadowmaster to be lowered to make it more rare, its damage resistance to non-gold weapons to be lowered and its darkness aura to be reduced in severity somewhat.
